fix(html): mention the subtitle code needs to be hosted (#34262)
mdn/content: files/en-us/learn/html/multimedia_and_embedding/video_and_audio_content/index.md

To get this displayed along with the HTML media playback, you need to:

> 1. Save it as a `.vtt` file near the HTML file.

Reviewer: ```suggestion
1. Save it as a `.vtt` file somewhere the server can serve (see below), such as in the same directory as the HTML file.
```

> 1. Save it as a `.vtt` file in the same directory where the `index.html` file is.

Reviewer: This is not a requirement. Writing it this way is confusing—why do we have this, anyway? The only requirement is that the server is able to serve it.

Author: I tried to align it with the corresponding code `<track kind="subtitles" src="subtitles_es.vtt" srclang="es" label="Spanish" />`. The `sensible` place is even more confusing to newbies, does sensible place mean "My Documents", "Desktop", or user home directory?

Author: ```suggestion
1. Save it as a `.vtt` file near the HTML file.
```

</video>
```

In order to try this you need to host the html file on a local server. You can use [`http-server`](https://www.npmjs.com/package/http-server) package. Put the code in an `index.html` file and run following command in the same directory where the html file is:

```bash
npx http-server -o
```

> 

Reviewer: The learn HTML section is the very first learning chapter, at which point readers might not have `npx`. I think it might be easier to link to https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Tools_and_setup/set_up_a_local_testing_server.

Author: Now I've provided link to the server setup page. A and on setup server page added a section about `npx http-server`.
